packages/client/package.json

```diff
@@ -35,5 +36,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@mdx-js/mdx": "^1.6.22",
+    "babel-loader": "^8.1.0"
```

Comment:
These aren't really dev dependencies since they're needed for the build. devDependencies are almost exclusively linting.